content/collections/browser_sdk/en/browser-sdk-2.md

```diff
@@ -1127,7 +1127,10 @@ amplitude.track("event")
 Unlike standard network requests, sendBeacon sends events in the background, even if the user closes the browser or leaves the page.
 
 {{partial:admonition type="warning" heading=""}}
-Because `sendBeacon` sends events in the background, events dispatched from `sendBeacon` don't return a server response and can't be retried when they encounter failures like 4xx or 5xx errors. You can address these retry issues by sending one event/request, but this could increase the network load and the likelihood of throttling.
+`sendBeacon` sends events in the background. As a result, events `sendBeacon` dispatches don't return a server response. Keep the following in mind if you use `sendBeacon`:
```

Comment:
Nit: Flows a bit better (subjective) and be consistent with plurality (events, dispatches, a server response). 
```suggestion
`sendBeacon` sends events in the background. As a result, events dispatched by `sendBeacon` don't return server responses. Keep the following in mind if you use `sendBeacon`:
```
or 
```suggestion
`sendBeacon` sends events in the background. As a result, events `sendBeacon` dispatches don't return server responses. Keep the following in mind if you use `sendBeacon`:```